operator/internal/manifests/config_test.go

```diff
@@ -489,3 +489,457 @@ func TestConfigOptions_RulerAlertManager_UserOverride(t *testing.T) {
+									Endpoints:      []string{"http://other-alertmanager"},
+									DiscoverySpec: &v1beta1.AlertManagerDiscoverySpec{
+										EnableSRV:       true,
+										RefreshInterval: "5m",
+									},
+									Client: &v1beta1.AlertManagerClientConfig{
+										TLS: &v1beta1.AlertManagerClientTLSConfig{
+											ServerName: pointer.String("other.svc"),
+											CAPath:     pointer.String("/tenant/other/alertmanager/ca.crt"),
+											CertPath:   pointer.String("/tenant/other/alertmanager/cert.crt"),
+											KeyPath:    pointer.String("/tenant/other/alertmanager/cert.key"),
+										},
+										BasicAuth: &v1beta1.AlertManagerClientBasicAuth{
+											Username: pointer.String("user"),
+											Password: pointer.String("pass"),
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				OpenShiftOptions: openshift.Options{
+					BuildOpts: openshift.BuildOptions{
+						AlertManagerEnabled:             true,
+						UserWorkloadAlertManagerEnabled: true,
+					},
+				},
+			},
+			wantOptions: map[string]config.LokiOverrides{
+				"application": {
+					Ruler: config.RulerOverrides{
+						AlertManager: &config.AlertManagerConfig{
+							ExternalURL:     "external",
+							Hosts:           "http://application-alertmanager",
+							EnableV2:        false,
+							EnableDiscovery: false,
+							RefreshInterval: "3m",
+							ExternalLabels:  map[string]string{"external": "label"},
+							Notifier: &config.NotifierConfig{
+								TLS: config.TLSConfig{
+									ServerName: pointer.String("application.svc"),
+									CAPath:     pointer.String("/tenant/application/alertmanager/ca.crt"),
+									CertPath:   pointer.String("/tenant/application/alertmanager/cert.crt"),
+									KeyPath:    pointer.String("/tenant/application/alertmanager/cert.key"),
+								},
+								HeaderAuth: config.HeaderAuth{
+									Type:        pointer.String("Bearer"),
+									Credentials: pointer.String("letmeinplz"),
+								},
+							},
+						},
+					},
+				},
+				"other-tenant": {
+					Ruler: config.RulerOverrides{
+						AlertManager: &config.AlertManagerConfig{
+							ExternalURL:     "external1",
+							Hosts:           "http://other-alertmanager",
+							EnableV2:        false,
+							EnableDiscovery: true,
+							RefreshInterval: "5m",
+							ExternalLabels:  map[string]string{"external1": "label1"},
+							Notifier: &config.NotifierConfig{
+								TLS: config.TLSConfig{
+									ServerName: pointer.String("other.svc"),
+									CAPath:     pointer.String("/tenant/other/alertmanager/ca.crt"),
+									CertPath:   pointer.String("/tenant/other/alertmanager/cert.crt"),
+									KeyPath:    pointer.String("/tenant/other/alertmanager/cert.key"),
+								},
+								BasicAuth: config.BasicAuth{
+									Username: pointer.String("user"),
+									Password: pointer.String("pass"),
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			desc: "openshift-network mode",
```

Comment:
Same applies here as in the above test `TestConfigOptions_RulerOverrides_OCPApplicationTenant`